app/views/api/umgap/casestudy_metagenomics.html.erb

```diff
@@ -409,8 +416,14 @@
           The <code>grep</code> command will drop the FASTA headers,
           allowing the <code>report</code> command to snap each taxon to either a species or "unknown"
           and report the frequency table of the results.
-          This command will prove more useful with larger input.
         </p>
+        <p>
+          The latter will send the results to the Unipept API, where an interactive HTML-visualisation will be rendered.
```

Comment:
`The <code>taxa2tree</code> command will send the results to the Unipept webserver where an interactive HTML-visualisation will be created.`